src/inmanta_ui/ui.py

```diff
@@ -80,6 +88,9 @@ def add_web_console_handler(self, server: Server) -> None:
         if json_parser_option == "BigInt":
             config_js_content += f"window.jsonParserId = '{json_parser_option}';\n"
 
+        features = "'" + "', '".join(web_console_features.get()) + "'"
```

Comment:
I would use a json encoder here or at least document what the intended end result is. 

(this will do  `export features = [``];`   is the option is not set)

Reply:
done